index.js

```diff
@@ -0,0 +1,61 @@
+// eslint-disable-next-line no-unused-vars 
+function validatePassword(password) { 
+
+  function length(password) {
+    if(password.length < 8) {
+      return false
+    }
+    else {
+      return true 
+    }
+  }
```

Comment:
We don't define functions inside of functions like this, it causes scope issues. They should be outside of `function validatePassword(password) { ` That's why you got the `no-unused-vars` error that you disabled. You should never need to disable that rule.

Reply:
Cleaned up the code to run efficiently